src/classes/ExpectDatetime.cls

```diff
@@ -0,0 +1,85 @@
+public class ExpectDatetime {
+    //probably we can use this string for the other implementations as well?
+    public static final String DATETIME_SHOULD_BE_MESSAGE = 'Expected {0} should{1} be {2}';
+    public static final String DATETIME_SHOULD_BE_LESS_MESSAGE = 'Expected {0} should be less than {1}';
+    public static final String DATETIME_SHOULD_BE_LESS_OR_EQUAL_MESSAGE = 'Expected {0} should be less than or equal to {1}';
+    public static final String DATETIME_SHOULD_BE_GREATER_MESSAGE = 'Expected {0} should be greater than {1}';
+    public static final String DATETIME_SHOULD_BE_GREATER_OR_EQUAL_MESSAGE = 'Expected {0} should be greater than or equal to {1}';
+    public static final String DATETIME_BETWEEN_MESSAGE = 'Expected {0} should be between {1} and {2}';
+
+    private final Datetime expected;
+
+    public ExpectDatetime(Datetime expected) {
+        this.expected = expected;
+    }
+    private ExpectDatetime assertThis(Boolean test, String message) {
+        Expect.assert(test, message);
+        return this;
+    }
+    public ExpectDatetime shouldEqual(Datetime actual) {
+        return shouldEqual(actual, String.format(DATETIME_SHOULD_BE_MESSAGE, new String[]{
```

Comment:
I think these String.format calls can be re-factored into the base Expect class with the formatMessage method I have in my branch?  I think it's nicer to have the static strings consolidated - your thoughts?